### PR TITLE
#2129 - Import of data info Menas fails on Windows 10

### LINKED ISSUE #2129: Import of data into Menas fails on Windows 10
## Describe the bug
Upload of json data files failing on `curl` command when opening json file. Used prepared script [`import_all.sh`](https://github.com/AbsaOSS/enceladus/blob/3a90c650654e0f123c5e1a21c0ecfb1b2f873867/examples/data/e2e_tests/setup_scripts/import_all.sh)
Problem in `curl` parameter `-d` when defined path to file using `@<path>`. 

Command:
```
curl -v -X POST 
  "http://localhost:8080/rest_api_war/api/schema/importItem" 
  -H "@/tmp/headers.txt.G7pBq7" 
  -H "Content-Type:application/json" 
  --data @/cygdrive/ .... /data/e2e_tests/data/exports/schemas/std_nf_dy.json
```

Output:
```
Warning: Couldn't read data from file
Warning: "/cygdrive/c/Users/ab024LL/absa/git/enceladus/examples/data/e2e_tests/
Warning: data/exports/datasets/uppercaseConfrormanceRule.json", this makes an
Warning: empty POST.
...
Required request body is missing: ....
```

## To Reproduce
Steps to reproduce the behavior OR commands run:
1. Expected Enceladus app built and running
2. Run script `import_all.sh` in your bash environment. (Clink, Cygwin)
3. Problem occurs, no data imported.

## Expected behavior
No error message visible. All data files imported in Menas DB.

## Desktop (please complete the following information):
 - OS: Windows 10 Enterprise 21H2 19044.1889
 - Clink 0.4.0 with curl 7.83.1
 - Cygwin with curl 7.83.1

## Additional context
Proposal to repair:
```
    for var in `ls -d ${PWD}/${1}/*`; do
        canonicalPath=$(realpath "${var}")
        fileContent=`cat ${canonicalPath}`                -- new variable
        curl -v \
            -H "@${HEADERS_FILE}" \
            -H 'Content-Type:application/json' \
            --data "${fileContent}" \                -- usage of variable, no @ used
            "${URL}/api/${2}/importItem"

```

### PR DESCRIPTION
**Pull request by miroslavpojer**
- Added new variable to store file content before send by curl command.
- Tested on Windows 10.

Closes #2129

### CODE REVIEW

**Review by dk1844 — APPROVED** [review]
Successfully ran on Windows 10 with Cygwin 👍

File: examples/data/e2e_tests/setup_scripts/import_all.sh [hunk 1]
```diff
@@ -45,10 +45,11 @@ fi
 import_function () {
     for var in `ls -d ${PWD}/${1}/*`; do
         canonicalPath=$(realpath "${var}")
+        fileContent=`cat ${canonicalPath}`
         curl -v \
             -H "@${HEADERS_FILE}" \
             -H 'Content-Type:application/json' \
-            --data "@${canonicalPath}" \
+            --data "${fileContent}" \
```
Comment (benedeki) [inline examples/data/e2e_tests/setup_scripts/import_all.sh:52]: What if the file content has some "powerful" characters like `"` or `\`?

**Review by lsulak — APPROVED** [review]
It works on my M1 Mac as well (when I add implementation of the 'realpath' function into the script manually since it's not available there - https://github.com/whatwg/html-build/issues/90 - I couldn't install it via `brew install coreutils`).

File: examples/data/e2e_tests/setup_scripts/import_all.sh (hunk 1 above)
Comment (miroslavpojer) [reply examples/data/e2e_tests/setup_scripts/import_all.sh:52]: Tested:
- input #1: `"hdfsPublishPath": "/publish/uppercaseConform\\anceRule"`
- imported #1: `"hdfsPublishPath":"/publish/uppercaseConform\\anceRule"`
- input #2: `"outputDataType": "st\"ring"`
- imported #2: `"outputDataType":"st\"ring"`

Comment (benedeki) [reply examples/data/e2e_tests/setup_scripts/import_all.sh:52]: Cool, thank you. 😎

**Review by benedeki — APPROVED** [review]

**Pull request merged**